service-automation/src/com/appyschool/tests/students/TestStudent.java

```diff
@@ -1,46 +1,60 @@
 package com.appyschool.tests.students;
 
 import java.util.HashMap;
-
+import java.util.Map;
 
 import org.apache.commons.lang3.RandomStringUtils;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.testng.annotations.Test;
 
+import com.appyschool.common.CommonMethods;
 import com.appyschool.common.CommonUtils;
+import com.appyschool.common.ConfigProperties;
 import com.appyschool.common.DataProviderArguments;
-import com.appyschool.services.BranchServices;
+import com.appyschool.common.GenericConstants;
+import com.appyschool.common.SoftAssertionBase;
+import com.appyschool.common.SuiteSetupProperties;
 import com.appyschool.services.StudentServices;
 import com.jayway.restassured.response.Response;
 
 public class TestStudent {
-
+	int n=0;
+	
 	/**
 	 * Create a Students
 	 * @param map
 	 * @throws JSONException
 	 */
-	@Test(dataProviderClass = com.appyschool.common.CommonUtils.class, dataProvider = "getDataFromFile")
-	@DataProviderArguments("./testData/csv/student.csv")
-	public void testCreateStudents(HashMap<String, String> map)
-			throws JSONException {
-		String jsonTemplate = CommonUtils
-				.getJsonTemplate("./testData/json/student.json");
+	@Test(dataProviderClass = com.appyschool.common.CommonUtils.class, dataProvider = "getDataFromFile" )
+	@DataProviderArguments("./testData/configuration/student.csv")
```

Comment:
where in code are you using student1.csv?

Make sure you don;t reuse the SuitSetup's student csv file, it will needlessly create lot of students.